src/components/ButtonItem.scss

```diff
@@ -4,6 +4,8 @@ $button-blue: #46cccc;
 $button-blue-hover: #3a9b94;
 $button-green: #46cc46;
 $button-green-hover: #359e37;
+$button-darkblue: #3a9b94;
```

Comment:
Både denne knappen og stylingen av den var bare en klargjøring for endringene som skal skje på Frontpage. Forventet at vi kom til å endre på det, derfor la jeg det bare inn i ButtonItem.scss. Ferdige btn-stylingen burde selvfølgelig være i _button.

Fargene kan vi godt legge inn i variables (eller bytte de ut med eksisterende farger). Er allerede gjort en del endringer på button items og frontpage av de som jobber med teacherInfoBox og welcomeBox, så kanskje greit at de gjør det